ui/admin/tests/acceptance/session-recordings/selectors.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+export const SESSION_RECORDING_TITLE = 'Session Recordings';
+export const TABLE_FIRST_ROW_ACTION_LINK =
```

Comment:
None blocking comment: do you think we can reuse [this common selector](https://github.com/hashicorp/boundary-ui/blob/main/ui/admin/tests/helpers/selectors.js#L24) rather than defining this specific one?

Reply:
This selector is clicking on a direct link at the end of the table row. The selector linked in your comment is clicking on a link inside a dropdown. We could argue that this new selector should go in the commonSelectors file but if it is only used for session recordings, we can leave it here.